common/tools/support.py

```diff
@@ -225,17 +227,19 @@ def check(ignore_missing):
 
 @cli.command()
 # fmt: off
-@click.option("--v1", help="Version for T1 release (default: guess from latest)")
-@click.option("--v2", help="Version for TT release (default: guess from latest)")
+@click.option("--t1b1", help="Version for T1 release (default: guess from latest)")
+@click.option("--t2t1", help="Version for TT release (default: guess from latest)")
+@click.option("--t2b1", help="Version for TR release (default: guess from latest)")
```

Comment:
please convert this to something like `-r T1B1=1.2.0 -r T2T1=2.6.7`, i.e., a multiple-option that specifies a key and a value

Reply:
Done in [16c0911](https://github.com/trezor/trezor-firmware/pull/3240/commits/16c09115560ee792d410394931d0b112ae394f53)